pkg/nodebootstrap/assets/simple.al2.userdata

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -o xtrace
+/etc/eks/bootstrap.sh ${ClusterName} ${BootstrapArguments}
```

Comment:
I this implies that you will have to bypass gzip+base64 encoding of user data... isn't it?

Reply:
`gzip+base64` is not useful when exported anyway